### PR TITLE
fix(i18n): update Chinese translation of basic css

### PR DESCRIPTION
**Pull request by ZhichengChen**
- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [X] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: curriculum/challenges/chinese/01-responsive-web-design/basic-css/add-a-negative-margin-to-an-element.chinese.md [hunk 1]
```diff
@@ -2,23 +2,30 @@
...
+<section id='description'>
+元素的<code>margin（外边距）</code>控制元素边框与其他周围元素之间的距离大小。
+如果你设置元素<code>margin</code>为负值，元素会变得更大。
+</section>
 
 ## Instructions
-<section id="instructions">尝试将<code>margin</code>设置为负值，如红色框的值。将蓝色框的<code>margin</code>更改为<code>-15px</code> ，因此它会填充其周围黄色框的整个水平宽度。 </section>
+<section id='instructions'>
+尝试设置蓝色盒子的<code>margin</code>为负值，跟红色盒子一样大小。
+蓝色盒子的<code>margin</code>设置为<code>-15px</code>，它会填满与黄色盒子之间的距离。
```
Comment (stevending1st) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-css/add-a-negative-margin-to-an-element.chinese.md]: “盒子”感觉比较牵强，还是“框”比较好？（后同）

File: curriculum/challenges/chinese/01-responsive-web-design/basic-css/add-a-negative-margin-to-an-element.chinese.md [hunk 2]
```diff
@@ -2,23 +2,30 @@
...
+videoUrl: 'https://scrimba.com/c/cnpyGs3'
+forumTopicId: 16166
+localTitle: 给元素添加负外边距
 ---
 
 ## Description
-<section id="description">元素的<code>margin</code>控制元素<code>border</code>与周围元素之间的空间量。如果将元素的<code>margin</code>设置为负值，则元素将变大。 </section>
+<section id='description'>
+元素的<code>margin（外边距）</code>控制元素边框与其他周围元素之间的距离大小。
+如果你设置元素<code>margin</code>为负值，元素会变得更大。
```
Comment (stevending1st) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-css/add-a-negative-margin-to-an-element.chinese.md]: 如果你把元素的<code>margin</code>设置为负值，元素会变得更大。

**Review by stevending1st — COMMENTED** [review]
2020/1/6 Stevending1st

**Review by miyaliu666 — APPROVED** [review]

**Comment by scissorsneedfoodtoo** [comment]
Thank you for all of your hard work updating these challenges, @ZhichengChen. Also, thank you @stevending1st and @miyaliu666 for your thorough review :+1: :+1:

**Pull request merged**